src/commands/remove.js

```diff
@@ -131,43 +134,57 @@ export default async function main(ctx) {
   const findStart = Date.now();
 
   try {
-    const searchFilter = (d) => ids.some((id) => (
-      (d && !(d instanceof NowError)) && (
-      d.uid === id ||
-      d.name === id ||
-      d.url === normalizeURL(id)
-    )));
+    const searchFilter = d =>
+      ids.some(
+        id =>
+          d &&
+          !(d instanceof NowError) &&
+          (d.uid === id || d.name === id || d.url === normalizeURL(id))
+      );
 
     const [deploymentList, projectList] = await Promise.all([
-      Promise.all(ids.map((idOrHost) => getDeploymentByIdOrHost(now, contextName, idOrHost))),
-      Promise.all(ids.map(async (idOrName) => getProjectByIdOrName(now, idOrName)))
+      Promise.all(
+        ids.map(idOrHost =>
+          getDeploymentByIdOrHost(client, contextName, idOrHost)
```

Comment:
This, and line 152 below are the main fix

Reply:
Can you add a test for this?